front/components/assistant/conversation/UserMessage.tsx

```diff
@@ -51,7 +51,7 @@ export function UserMessage({
     <div className="flex flex-grow flex-col">
       <div className="max-w-full self-end">
         <ConversationMessage
-          pictureUrl={message.user?.image || message.context.profilePictureUrl}
+          pictureUrl={message.context.profilePictureUrl || message.user?.image}
```

Comment:
Change of behavior for all UserMessages but this is the behavior that makes sense.